plugins/node/instrumentation-cucumber/package.json

```diff
@@ -43,7 +43,7 @@
     "@opentelemetry/api": "^1.0.0"
   },
   "devDependencies": {
-    "@cucumber/cucumber": "^9.0.0",
+    "@cucumber/cucumber": "^10.0.0",
```

Comment:
Given cucumber v10 dropped Node.js v14 and v16 support that this package still supports, I think we want to keep the version here on cucumber v9. *Currently* `@cucumber/cucumber@10.0.1` seems to work with tests with Node.js v14 and v16, but that could break later.